go/database/mpt/live_trie.go

```diff
@@ -60,18 +60,36 @@ func OpenFileLiveTrie(directory string, config MptConfig, cacheCapacity int) (*L
 	return makeTrie(directory, forest)
 }
 
+// VerifyMptStateWithLiveTrie validates Mpt state with file-based live trie stored
+// in the given directory. If the test passes, the data stored in the respective
+// directory can be considered to be a valid Mpt state of the given configuration.
+// This validation contains both Forest and contract codes checks.
+func VerifyMptStateWithLiveTrie(directory string, config MptConfig, observer VerificationObserver) error {
```

Comment:
... also here, it looks like the name just chains all terms we have into a method name